index.html

```diff
@@ -58,7 +60,221 @@ <h3>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+    <div class="work-2">
+        <div class="proj-work">
+
+       
+        <section class="projects">
+            Profesional Art Printing Data
+         </section>
+         <section class="projects-details">
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+    <div class="work-2">
+        <div class="proj-work">
+
+       
+        <section class="projects">
+            Profesional Art Printing Data
+         </section>
+         <section class="projects-details">
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+    <div class="work-2">
+        <div class="proj-work">
+
+       
+        <section class="projects">
+            Profesional Art Printing Data
+         </section>
+         <section class="projects-details">
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+    <div class="work-2">
+        <div class="proj-work">
+
+       
+        <section class="projects">
+            Profesional Art Printing Data
+         </section>
+         <section class="projects-details">
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+  </section>
 </main>
 
+<div class="about">
```

Comment:
- [ ] I think it's better to use the HTML semantic tag `<section>` instead of a div. To make your project more semantically correct.